src/neuroconv/tools/spikeinterface/spikeinterface.py

```diff
@@ -1072,13 +1072,16 @@ def get_nspikes(units_table: pynwb.misc.Units, unit_id: int):
 def add_units_table(
     sorting: SpikeInterfaceSorting,
     nwbfile: pynwb.NWBFile,
+    unit_ids: Optional[List[Union[str, int]]] = None,
     property_descriptions: Optional[dict] = None,
     skip_properties: Optional[List[str]] = None,
     skip_features: Optional[List[str]] = None,
     units_table_name: str = "units",
     unit_table_description: str = "Autogenerated by neuroconv.",
     write_in_processing_module: bool = False,
     write_waveforms: bool = False,
+    waveform_means: Optional[np.array] = None,
+    waveform_sds: Optional[np.array] = None,
```

Comment:
```suggestion
    waveform_means: Optional[np.ndarray] = None,
    waveform_sds: Optional[np.ndarray] = None,
```

or otherwise define generally as the `ArrayType` from `neuroconv.utils`